internal/config/docs.toml

```diff
@@ -0,0 +1,624 @@
+#
+# The interval in which the RegistrySynchronizer performs a full sync of the keeper registry contract it is tracking.
+RegistrySyncInterval = '30m' # Default
+# **ADVANCED**
+#
+# Do not change this setting unless you know what you are doing.
+#
+# Represents the maximum number of upkeeps that can be synced in parallel.
+RegistrySyncUpkeepQueueSize = 10 # Default
+# The number of blocks in the past to look back when getting a block for a turn.
+TurnLookBack = 1000 # Default
+# Enables a new algorithm for how keepers take turns.
+TurnFlagEnabled = false # Default
+# Use this setting *only* on Polygon networks.
+#
+# Includes gas price in calls to `checkUpkeep()` when set to `true`.
+UpkeepCheckGasPriceEnabled = false # Default
+
+# The Chainlink node is equipped with an internal "nurse" service that can perform automatic `pprof` profiling when the certain resource thresholds are exceeded, such as memory and goroutine count. These profiles are saved to disk to facilitate fine-grained debugging of performance-related issues. In general, if you notice that your node has begun to accumulate profiles, forward them to the Chainlink team.
+#
+# To learn more about these profiles, read the [Profiling Go programs with pprof](https://jvns.ca/blog/2017/09/24/profiling-go-with-pprof/) guide.
+[AutoPprof]
+# Set to `true` to enable the automatic profiling service.
+Enabled = false # Default
+# The location on disk where pprof profiles will be stored. Defaults to `RootDir`.
+ProfileRoot = 'prof/root' # Example
+# The interval at which the node's resources are checked.
+PollInterval = '10s' # Default
+# The duration for which profiles are gathered when profiling starts.
+GatherDuration = '10s' # Default
+# The duration for which traces are gathered when profiling is kicked off. This is separately configurable because traces are significantly larger than other types of profiles.
+GatherTraceDuration = '5s' # Default
+# The maximum amount of disk space that profiles may consume before profiling is disabled.
+MaxProfileSize = '100mb' # Default
+# See https://pkg.go.dev/runtime#SetCPUProfileRate.
+CPUProfileRate = 1 # Default
+# See https://pkg.go.dev/runtime#pkg-variables.
+MemProfileRate = 1 # Default
+# See See https://pkg.go.dev/runtime#SetBlockProfileRate.
+BlockProfileRate = 1 # Default
+# See https://pkg.go.dev/runtime#SetMutexProfileFraction.
+MutexProfileFraction = 1 # Default
+# The maximum amount of memory the node can actively consume before profiling begins.
+MemThreshold = '4gb' # Default
+# The maximum number of actively-running goroutines the node can spawn before profiling begins.
+GoroutineThreshold = 5000 # Default
+
+[Sentry]
+Debug = false # Default
+DSN = 'sentry-dsn'
+Environment = 'dev'
+Release = 'v1.2.3'
+
+[[EVM]]
+ChainID = '1'
+Enabled = true # Default
+BalanceMonitorEnabled = true
+BlockBackfillDepth = 100
+BlockBackfillSkip = true
+BlockEmissionIdleWarningThreshold = '1h'
```

Comment:
I think this can be grouped under "HeadTracker" if that section exists.

```
BlockEmissionIdleWarningThreshold will cause Chainlink to log warnings if this duration is exceeded without any new blocks being emitted.
```

Reply:
Not yet, but this would be the fourth field, which seems like enough to separate as another table :+1: